cmd/brew-command-not-found-init-fish.rb

```diff
@@ -0,0 +1,7 @@
+
+require "fileutils"
+
+file = File.expand_path("#{File.dirname(__FILE__)}/../handler.fish")
+destination = ARGV[0] ? ARGV[0]+"command-not-found.fish" : File.expand_path("~/.config/fish/functions/command-not-found.fish")
+
+FileUtils.cp(file, destination)
```

Comment:
I’d prefer if we could have only one command, we can use an option e.g.: `brew command-not-found-init --fish`.